runtime-modules/working-group/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = 'substrate-working-group-module'
-version = '1.0.1'
+version = '1.0.2'
```

Comment:
I'm going to be picky with the version numbers here because we are not publishing them, but technically the change although a "bug fix", is actually a change in behavior

Reply:
Ok